### PR TITLE
[check-redis] Add password support for check-redis

### PR DESCRIPTION
**Pull request by n-rodriguez**
Thank you!

### CODE REVIEW

File: check-redis/lib/check-redis.go [hunk 1]
```diff
@@ -63,6 +64,14 @@ func connectRedis(m redisSetting) (*redis.Client, error) {
 	if err != nil {
 		return nil, fmt.Errorf("couldn't connect: %s", err)
 	}
+
+	if m.Password != "" {
+		r := c.Cmd("AUTH", m.Password)
+		if r.Err != nil {
+			return nil, errors.New("couldn't execute query")
```
Comment (lufia) [inline check-redis/lib/check-redis.go]: Do this *r.Err* contain the case of authentication failed by password incorrect? If so this error message is not kindly. I think it is better that an error includes an error from Redis, like `fmt.Errorf("couldn't authenticate: %v", r.Err)`.

Comment (n-rodriguez) [reply check-redis/lib/check-redis.go]: >  I think it is better that an error includes an error from Redis

Right! Fixed :)

**Comment by lufia** [comment]
Thank you! We will review this p-r.

**Comment by n-rodriguez** [comment]
Thank you!

**Pull request merged**